### PR TITLE
New version: JetBrains.Gateway version 2022.2.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | JetBrains Gateway | CLion 223.6160.115    |
| Version     | 2022.2.3     | 223.6160.115 |
| Publisher   | JetBrains s.r.o.   | JetBrains s.r.o.      |
| ProductCode |           | CLion 223.6160.115    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82732)

### CODE REVIEW

File: manifests/j/JetBrains/Gateway/2022.2.3/JetBrains.Gateway.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: JetBrains.Gateway
+PackageVersion: 2022.2.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+ReleaseDate: 2022-10-06
```
Comment (mdanish-kh) [inline manifests/j/JetBrains/Gateway/2022.2.3/JetBrains.Gateway.installer.yaml:9]:
```suggestion
ReleaseDate: 2022-10-06
UpgradeBehavior: uninstallPrevious
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by zachcarp** [comment]
@vedantmgoyal2009 This appears to just hang for me when I run it, and it looks like the same issue is happening in the automated process.

**Comment by zachcarp** [comment]
@ItzLevvie - Did you have any issues running this? Maybe its an architecture issue on my end?

**Pull request closed without merging**